src/MCPClient/lib/clientScripts/archivematicaCreateMETS2.py

```diff
@@ -920,9 +964,17 @@ def create_object_metadata(struct_map, baseDirectoryPath):
     fileGroupType = opts.fileGroupType
     includeAmdSec = opts.amdSec
 
-    while False: #used to stall the mcp and stop the client for testing this module
-        import time
-        time.sleep(10)
+    # If reingesting, do not create a new METS, just modify existing one
+    if 'REIN' in SIP_TYPE:
```

Comment:
SIP_TYPE can be `AIP`, `AIC`, `AIP-REIN` or `AIC-REIN`, though I don't think we support reingesting AICs properly right now.